tests/test_fx/test_pipeline/test_topo/test_topo.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import torch
+import transformers
+from topo_utils import split_model_and_get_DAG, check_topo, MLP
+
+BATCH_SIZE = 1
+SEQ_LENGHT = 16
+
+
+@pytest.mark.skip('balance split v2 is not ready')
```

Comment:
I think this part of unit tests should be decoupled from fx split passes.

Reply:
It's a test mainly for fx split combined with Topo. I think maybe a new test case should be added, which is a simple one just for Topo itself.